staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/conversion.go

```diff
@@ -41,15 +41,22 @@ func Convert_apiextensions_JSON_To_v1beta1_JSON(in *apiextensions.JSON, out *JSO
 	if err != nil {
 		return err
 	}
-	out.Raw = raw
+	if len(raw) == 0 || string(raw) == "null" {
+		// match JSON#UnmarshalJSON treatment of literal nulls
+		out.Raw = nil
+	} else {
+		out.Raw = raw
+	}
 	return nil
 }
 
 func Convert_v1beta1_JSON_To_apiextensions_JSON(in *JSON, out *apiextensions.JSON, s conversion.Scope) error {
 	if in != nil {
 		var i interface{}
-		if err := json.Unmarshal(in.Raw, &i); err != nil {
-			return err
+		if len(in.Raw) > 0 && string(in.Raw) != "null" {
```

Comment:
Avoid the string temporary.

Reply:
ack, will fix in JSON#UnmarshalJSON as well

Reply:
switched to comparing using bytes.Equal to avoid allocation and short-circuit on length mismatch